### ISSUE TITLE
PDF title and Browser title shows weird characters 

### ISSUE THREAD

**Issue by awaisijaz1994** [dompdf/dompdf#2745]
I am using dompdf for generating the PDFs.  I am facing an issue regarding weird characters in my PDF's title and name on maximum browsers like chrome, firefox, edge and brave. While generating the pdf I set the title as PDF (Shown below).
```
<head>
    <meta charset="utf-8">
    <title>PDF</title>
  </head>
```
But on the browser's pdf viewers it shows like that

**PDF힢㭀寗텠⅖鼄�壽똓㈥徫鰬喵꧄⼂뎺䋄ቍٌ臗ퟁ喞㦴耖ʆ♥슉㊑㶉௫銠᫩ꉥ䠾賖蹩턦艜**

Most of the PDF don't have this issue. some of them have this issue. Moreover, it just shows on the browser not if a pdf file is already downloaded to the system. I will add some images for better understanding.

looking forward to some response.
Thank you

![chrome pdf issue2](https://user-images.githubusercontent.com/52527529/149932296-b4e3bbd2-2aa2-4ec0-a059-8aa61305c837.PNG)
![chrome pdf issue](https://user-images.githubusercontent.com/52527529/149932302-d584e683-0f0c-4553-aa14-1eaf509d902a.PNG)

**Comment by Mellthas** [comment]
This might be related to or even the same as #1541. Can you share a sample document and related PHP code? In particular, it would be good to know if you have encryption enabled.

**Comment by awaisijaz1994** [comment]
yes i am using encryption in my PDF
// instantiate and use the dompdf class
    $dompdf = new Dompdf($options);

    $dompdf->loadHtml($html);

// (Optional) Setup the paper size and orientation
    $dompdf->setPaper('A4', 'portrait');

//    $customPaper = array(0, 0, 900, 900);
    $customPaper = array(0,0,612.00,792.00);
    $dompdf->set_paper($customPaper);
// Render the HTML as PDF
    $dompdf->render();

$dompdf->get_canvas()->get_cpdf()->setEncryption('','password here',array('copy','print'));

**Comment by Mellthas** [comment]
Then the issue is fixed on master, see #2683, next release should be out soon. Closing as a duplicate of #1541. If you still encounter the issue after updating, please reopen with more information.
